### ISSUE TITLE
"Colors" tab missing from mate-appearance-properties for "TraditionalGreen" and "TraditionalOk" themes

### ISSUE THREAD

**Issue by rubberbandaid** [mate-desktop/mate-control-center#419]
Without this "Colors" tab, the colors cannot be customized. Most importantly, the top color bar for selected windows cannot be changed.

#### Expected behaviour

I will demonstrate by example. Github won't let me upload the images, so I'm putting them on imgur, a well known image hosting site.
Here is a Ubuntu MATE 16.04.2 live CD environment, which has the "Colors" tab, as expected:
https://imgur.com/a/trkEmpS
Here is a Ubuntu MATE 18.04 live CD environment, which does not have the "Colors" tab, as it should:
https://imgur.com/a/xWW9XaP

#### Actual behaviour

"Colors" tab that used to exist is gone. No way to set the color of the top bar for selected windows, which is useful for separating virtual machines visually.

#### Steps to reproduce the behaviour

1. Run a live CD of Ubuntu MATE 16.04.2
2. Open mate-appearance-properties
3. Select "TraditionalGreen" or "TraditionalOk"
4. Hit "Customize", and note the presence of the "Colors" tab.

Then, repeat the same steps for Ubuntu MATE 18.04, and the "Colors" tab will be missing.

I love MATE, thanks for all the hard work everyone.

**Comment by rubberbandaid** [comment]
This being missing could be somehow related to this issue in Linux Mint, where the window icon and color are not set, and a selected window looks the same as an unselected window in Linux Mint:
https://imgur.com/a/YjLaF2H
The first picture is based on 16.04, the second picture is based on 18.04.

**Comment by lukefromdc** [comment]
As I recall, the "colors" tab was removed because it no longer worked on GTK 3. GTK2 and GTK3 theming are entirely different. To recolor a GTK3 theme that includes any image files with the color in question included might not even be possible. The theme would have to be written from scratch to support being recolored, carefully avoiding the use of any image files and making all images from symbolic colors defined at the start of the theme and thus overrideable from ~/.config.gtk.css. Might even be that that would be the only place such color overrrides could be stored. 

In other words, a color tab would be a total do-over from scratch, and only work with themes written explicitly to support it. Probably easier to (as is done in mate-themes ) have multiple hardcoded color varients of a theme.

**Comment by lukefromdc** [comment]
Note that Linuxmint (unlike their Ubuntu base) kept build MATE 1.16  against GTK2, which allowed the colors tab to work for one more cycle. MATE 1.16 was the last version to support GTK2.

**Comment by rubberbandaid** [comment]
Thanks for the explanation Luke. Makes sense to leave things as is then. I'll close this issue.
